week-2-wide-perspective/w02e03-pagination-problem/src/middleware.js

```diff
@@ -0,0 +1,22 @@
+export const  paginate = ({skipParamName, limitParamName}) => (req, res, next) => {
+  const skip = req[skipParamName]
```

Comment:
yyy sprawdzałeś czy to działa finalnie?

Bo mnie tutaj brauje `req.query` na zasadzie:
```
req.query[skipParamName]
```

Kozacko by było jakby dało się określić w Twoim @WojciechJelen `config`u - skąd m wyciągnąć te paramsy, a domyślnie byłoby to `query`